pkg/distributor/http.go

```diff
@@ -34,6 +34,11 @@ func (d *Distributor) PushHandler(w http.ResponseWriter, r *http.Request) {
 				"err", err,
 			)
 		}
+
+		if d.tenantConfigs.LogWriteFailures(tenantID) {
```

Comment:
Could we push this down into the write manager, so it determines whether it should log?

Reply:
Agreed, it makes this leaner (on Distributor HTTP code). Done.